examples/with-create-react-app/src/index.tsx

```diff
@@ -0,0 +1,57 @@
+import "@rainbow-me/rainbowkit/styles.css";
+import "./polyfills";
+import "./index.css";
+import React from "react";
+import ReactDOM from "react-dom/client";
+import reportWebVitals from "./reportWebVitals";
+import { getDefaultConfig, RainbowKitProvider } from "@rainbow-me/rainbowkit";
+import { WagmiProvider } from "wagmi";
+import {
+  arbitrum,
+  base,
+  mainnet,
+  optimism,
+  polygon,
+  sepolia,
+  zora,
+} from "wagmi/chains";
+
+import App from "./App";
+import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
+
+const config = getDefaultConfig({
+  appName: "RainbowKit demo",
+  projectId: "YOUR_PROJECT_ID",
```

Comment:
@KosmosKey This in-line usage is nice; let's use that everywhere across the examples